src/main/webapp/view/video/video.js

```diff
@@ -50,6 +48,11 @@ export default class Video {
     startVideoSyncTimer();
   }
 
+  /** Returns current video time of 'videoPlayer'. */
+  getCurrentVideoTimeSeconds() {
```

Comment:
Seems error prone to have some times be in ms and some be sec. Shouldn't everything be in ms?

Reply:
This is being fixed by @n-wach